db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -377,16 +383,18 @@ consolidateEpochBlocks fpath xs = ExceptT $ do
             (liftIO . hClose)
             (\hdl -> do
                 liftIO $ BS.hPutStr hdl epochFileHeader
-                mapM (consolidate hdl) xs
+                mapM (consolidate hdl) $ zip [0 .. ] xs
                 )
     pure $ case partitionEithers ys of
             ([], zs) -> Right $ epochIndexToOffset zs
             (e:_, _) -> Left e
   where
     consolidate
         :: ConsolidateM ctx m
-        => Handle -> SlotIndexHash -> m (Either ConsolidateError SlotIndexLength)
-    consolidate hdl  (SlotIndexHash lsi hh) = do
+        => Handle -> (Int, SlotIndexHash) -> m (Either ConsolidateError SlotIndexLength)
+    consolidate hdl  (indx, SlotIndexHash lsi hh) = do
+        when (indx `mod` 1000 == 0) $
+            sleepSeconds 2
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Block/Epoch.hs#L386-L397